flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStKeyedStateBackend.java

```diff
@@ -510,6 +519,14 @@ public void dispose() {
             if (this.disposed) {
                 return;
             }
+            // This call will block until all clients that still acquire access to the ForSt
```

Comment:
nits: 

- do you mean: 
still acquire access -> still have the lock 
- the comment formatting is strange - it is split of many lines.
- maybe change "so that we cannot release the native resources while clients are still working with it in parallel." to "so that we do not release the native resources while clients (running on different threads) are still working with it"

Reply:
> still acquire access -> still have the lock

no, it's about `resource guard`, not `lock`.

> the comment formatting is strange - it is split of many lines.

The spotless did this format.